### ISSUE TITLE
Should the ios-swift.enum format have 'public' keyword

### ISSUE THREAD

**Issue by dbanksdesign** [amzn/style-dictionary#295]
⚠️I am not a Swift Developer so I have no idea what I'm talking about ⚠️

The actual enum values don't have 'public' keyword:
https://github.com/amzn/style-dictionary/blob/master/lib/common/templates/ios-swift/enum.swift.template#L37

Do the values need the 'public' keyword, or is there a case to make those keywords configurable like:

```js
{
  destination: 'enum.swift',
  format: 'ios-swift/enum.swift',
  keywords: 'public static'
}
```

iOS/swift devs please discuss.

**Comment by rickycpadilla** [comment]
@dbanksdesign Yes, they do need to have public access level. See my PR :)

**Comment by dbanksdesign** [comment]
Fixed in #316
